libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -358,10 +358,26 @@ bool ESP8266WiFiSTAClass::reconnect() {
  * @return  one value of wl_status_t enum
  */
 bool ESP8266WiFiSTAClass::disconnect(bool wifioff) {
+    return disconnect(wifioff, false);

```

Comment:
Default behaviour was always `true` no ?

Reply:
Yes, it was. Good point.